test/jdk/java/awt/Choice/ChoiceDragEventsInside/ChoiceDragEventsInside.java

```diff
@@ -0,0 +1,235 @@
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 6251983 6722236
+ * @summary MouseDragged events not triggered for Choice when dragging it with left mouse button
+ * @key headful
+ * @run main ChoiceDragEventsInside
+ */
+
+import java.awt.Choice;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.EventQueue;
+import java.awt.FlowLayout;
+import java.awt.Frame;
+import java.awt.Point;
+import java.awt.Robot;
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseMotionAdapter;
+import java.lang.reflect.InvocationTargetException;
+
+public class ChoiceDragEventsInside extends Frame {
+    Robot robot;
+    Choice choice1;
+    Point pt;
+    Dimension size;
+    volatile boolean mouseDragged = false;
+    volatile boolean mouseDraggedOutside = false;
+
+    public void setupUI() {
+        choice1 = new Choice();
+        for (int i = 1; i < 50; i++) {
+            choice1.add("item-0" + i);
+        }
+        choice1.setForeground(Color.red);
+        choice1.setBackground(Color.red);
+        choice1.addMouseMotionListener(new MouseMotionAdapter() {
+                public void mouseMoved(MouseEvent me) {
+                    System.out.println(me);
+                }
+
+                public void mouseDragged(MouseEvent me) {
+                    System.out.println(me);
+                    mouseDragged = true;
+                    if (me.getY() < 0) {
+                        mouseDraggedOutside = true;
+                    }
+                }
+            }
+        );
+        add(choice1);
+        setLayout(new FlowLayout());
+        setSize(200, 200);
+        setLocationRelativeTo(null);
+        setVisible(true);
+        validate();
+    }
+
+    public void start() {
+        try {
+            robot = new Robot();
+            robot.setAutoWaitForIdle(true);
+            robot.setAutoDelay(50);
+            robot.delay(100);
+            EventQueue.invokeAndWait(() -> {
+                pt = choice1.getLocationOnScreen();
+                size = choice1.getSize();
+            });
+            testDragInsideChoice(InputEvent.BUTTON1_MASK);
+            //            testDragInsideChoiceWhileUnfurled(InputEvent.BUTTON1_MASK);
```

Comment:
Why is this commented out line present ?
Should it be uncommented ?
Otherwise why do you have a method definition of this otherwise uncalled method ?

Reply:
That part of the test passes on Linux and fails on macOS and Windows. I think it is better to remove this test scenario completely because this behavior is not documented and it is platform specific.